README.md

```diff
@@ -295,3 +339,34 @@ git push --follow-tags
 # Commercial support
 
 Checkout https://adorsys.com/en/products/keycloak-config-cli/ for commercial support.
+
```

Comment:
please refer to the https://github.com/adorsys/keycloak-config-cli/blob/main/docs/MANAGED.md for resource management